source/features/latest-tag-button.tsx

```diff
@@ -101,7 +101,7 @@ async function init(): Promise<false | void> {
 
 	const link = (
 		<a
-			className="btn btn-sm btn-outline ml-0 flex-self-center css-truncate rgh-latest-tag-button"
+			className="btn btn-sm ml-0 flex-self-center css-truncate rgh-latest-tag-button"
```

Comment:
Most buttons on GitHub and Refined GitHub created don't use `.btn-outline`, and they feels out of place with the blueish appearance. I only left one usage in `follow-file-renames` because GitHub's button for that one has `.btn-outline`.

Reply:
Indeed it looks like they recently changed it.

# Before

![highlight-file-in-prs](https://user-images.githubusercontent.com/55841/60622834-879e1f00-9de1-11e9-9a9e-bae5ec0b3728.png)

## Now

<img width="266" alt="Screen Shot 3" src="https://user-images.githubusercontent.com/1402241/132982690-13b5575c-2d12-4995-8282-7a337fab8d69.png">